### DISCUSSION TITLE
Sunken Chest

### DISCUSSION THREAD

**Discussion by Krisshawnee** [blish-hud/Blish-HUD#973] [Requested Marker Packs]
Farming route for Sunken Chests with a starting point and auto copying of waypoints for the next one if possible. It would make it much less irritating to use up excess keys. I found where someone had attempted to make one, but it looks... out dated maybe? Some of the chests aren't where the route indicates.

**Comment by Metallis** [comment]
Apologies for the delay, this has been added with the new chest locations in the (MMM) Metal Marker Myriad pack for a while now. There is a starting location next to the NPC in LA that can copy all the WPs for you. Each chest also has an invisible marker that will copy the next closest one in the chain, but these don't start from LA, they start at one "end" and progress to the other.
